packages/web-components/src/text/text.stories.ts

```diff
@@ -0,0 +1,119 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../__test__/helpers.js';
+import type { Text as FluentText } from './text.js';
+import './define.js';
+
+type TextStoryArgs = Args & FluentText;
+type TextStoryMeta = Meta<TextStoryArgs>;
+
+/**
+ * Used to generate slotted content for stories
+ * @param as - the element to generate
+ * @param content - the content for the element
+ * @returns ViewTemplate
+ */
+const generateSemanticElementTemplate = (as: string, content) => {
```

Comment:
I would expect this to be the part of the `text` component's template - so that `<fluent-text>hi</fluent-text>` renders
```html
<:host>
  <p>
    <slot>hi</slot>
  </p>
</:host>
```